OHS-Hosting-Infrastructure/complaint-tracker
# Terraform for RDS

## Description of change

Terraform code for creating RDS instance that will bind to the Rails application


## Acceptance Criteria

- [ ] creates RDS instance
- [ ] customizable for multiple environments
- [ ] well organized Terraform code, everybody is happy
- [ ] no secret / passwords rattling around in the world at large

## How to test

Otherwise, follow the instructions in `terraform/README.md` to make sure that Terraform is running correctly and can set up RDS in your sandbox.  You won't actually be able to bind anything to your application unless if you also set up rails running in your cloud.gov sandbox....I did not write down those instructions but can add them here if that would be helpful.


## Issue(s)

* closes https://github.com/OHS-Hosting-Infrastructure/complaint-tracker/issues/87


## Checklist

To be completed by the submitter:



## To the Reviewer

This project is using [Conventional Comments](https://conventionalcomments.org/) to give structure
and context to PR comments. Please use these labels in your comments.

* **praise:**
* **nitpick:**
* **suggestion:**
* **issue:**
* **question:**
* **thought:**
* **chore:**
